infra/aws/terraform/prow-build-cluster/Makefile

```diff
@@ -15,6 +15,8 @@
 TF ?= terraform
 TF_ARGS ?=
 ASSUME_ROLE ?= true
+
+# https://github.com/hashicorp/terraform-provider-kubernetes-alpha/issues/199#issuecomment-832614387
```

Comment:
Can we add a TODO here that we might want to consider some other approach? I think @sftim mentioned that it's possible to workaround this.